### PR TITLE
ehr metrics: unify ambiguous term `site` as organizations

### PR DESCRIPTION
**Pull request by rennat**
In the original work specification for the EHR metrics API, the term `site` was interpreted in different ways by different stakeholders.
After discussions, we decided that these APIs should work only on `Organization`.

### CODE REVIEW

**Comment by rennat** [comment]
@wangy70 This change will impact your work on the public metrics api. I will look at it.

**Comment by wangy70** [comment]
> @wangy70 This change will impact your work on the public metrics api. I will look at it.

**Comment by wangy70** [comment]
> @wangy70 This change will impact your work on the public metrics api. I will look at it.

Thanks, let's talk about it on Monday.

File: rest-api/alembic/versions/ac995f0b54b8_ehr_metrics_use_organizations.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('ehr_receipt', sa.Column('organization_id', sa.Integer(), nullable=False))
+    op.drop_constraint(u'ehr_receipt_ibfk_1', 'ehr_receipt', type_='foreignkey')
```
Comment (wangy70) [inline rest-api/alembic/versions/ac995f0b54b8_ehr_metrics_use_organizations.py]: Do you need the columns in a specified order? If yes, we may need to specify the order by using `AFTER`.

Comment (rennat) [reply rest-api/alembic/versions/ac995f0b54b8_ehr_metrics_use_organizations.py]: No the column order does not matter for this table.

**Pull request merged**